py3status/modules/backlight.py

```diff
@@ -36,12 +36,14 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     color = '#FFFFFF'
+    device = None
     device_path = None
```

Comment:
Only config parameters should be defined here.

the ideal thing would be to add a post_config_hook method (this gets called once the module has had its config set but before its output method is first called.
```
def post_config_hook(self):
    if not self.device:
        self.device_path = get_device_path()
    else:
        self.device_path = "/sys/class/backlight/%s" % self.device
```